packages/sdk/src/constants.ts

```diff
@@ -24,5 +24,5 @@
 
  */
 
-export const sdkVersion = '21.0'
+export const sdkVersion = '21.1'
```

Comment:
We need to regen against 21.0 or edit to 21.0

Reply:
Thanks for catching. Learning more and more about this process